element_array_ephys/ephys_organoids.py

```diff
@@ -128,382 +130,178 @@ class AcquisitionSoftware(dj.Lookup):
-            file_path (varchar(255) ): relative file path for electrophysiology recording.
-        """
-
-        definition = """
-        # Paths of files of a given EphysRecording round.
-        -> master
-        file_path: varchar(255)  # filepath relative to root data directory
-        """
-
     def make(self, key):
-        """Populates table with electrophysiology recording information."""
-        session_dir = find_full_path(
-            get_ephys_root_data_dir(), get_session_directory(key)
-        )
-
-        inserted_probe_serial_number = (ProbeInsertion * probe.Probe & key).fetch1(
-            "probe"
-        )
-
-        # search session dir and determine acquisition software
-        for ephys_pattern, ephys_acq_type in (
-            ("*.ap.meta", "SpikeGLX"),
-            ("*.oebin", "Open Ephys"),
-        ):
-            ephys_meta_filepaths = list(session_dir.rglob(ephys_pattern))
-            if ephys_meta_filepaths:
-                acq_software = ephys_acq_type
-                break
-        else:
-            raise FileNotFoundError(
-                f"Ephys recording data not found!"
-                f" Neither SpikeGLX nor Open Ephys recording files found"
-                f" in {session_dir}"
-            )
-
-        supported_probe_types = probe.ProbeType.fetch("probe_type")
-
-        if acq_software == "SpikeGLX":
-            for meta_filepath in ephys_meta_filepaths:
-                spikeglx_meta = spikeglx.SpikeGLXMeta(meta_filepath)
-                if str(spikeglx_meta.probe_SN) == inserted_probe_serial_number:
-                    break
-            else:
-                raise FileNotFoundError(
-                    f"No SpikeGLX data found for probe insertion: {key}"
-                    + " The probe serial number does not match."
-                )
-
-            if spikeglx_meta.probe_model in supported_probe_types:
-                probe_type = spikeglx_meta.probe_model
-                electrode_query = probe.ProbeType.Electrode & {"probe_type": probe_type}
-
-                probe_electrodes = {
-                    (shank, shank_col, shank_row): key
-                    for key, shank, shank_col, shank_row in zip(
-                        *electrode_query.fetch("KEY", "shank", "shank_col", "shank_row")
-                    )
-                }
-
-                electrode_group_members = [
-                    probe_electrodes[(shank, shank_col, shank_row)]
-                    for shank, shank_col, shank_row, _ in spikeglx_meta.shankmap["data"]
-                ]
-            else:
-                raise NotImplementedError(
-                    "Processing for neuropixels probe model"
-                    " {} not yet implemented".format(spikeglx_meta.probe_model)
-                )
+        subject_id = key["induction_id"]
+        data_dir = get_ephys_root_data_dir()[0] / subject_id
+        data_files = data_dir.glob(f"{subject_id}*.rhs")
```

Comment:
@kabilar I know we talked about searching from session directory but I'm not sure about it since this table depends on subject table.

Reply:
Since we are recording continuously and storing the data within `root_directory > subject_id > session_id > probe_id`, where `session_id` stores the data from a single day, I think it will now work to search within the `subject_id` directory for all `*.rhs` files to enter into this table.  We can use the following function which is defined in the workflow to get the subject directory (although the name suggests otherwise):
```
session_dir = find_full_path(get_ephys_root_data_dir(), get_session_directory(key)
```